bench-tps/src/cli.rs

```diff
@@ -353,6 +357,24 @@ pub fn build_args<'a>(version: &'_ str) -> App<'a, '_> {
                 .validator(|arg| is_within_range(arg, 1..))
                 .help("The number of unique destination accounts per transactions 'chunk'. Lower values will result in more transaction conflicts.")
         )
+        .arg(
+            Arg::with_name("bind_address")
+                .long("bind-address")
+                .value_name("HOST")
+                .takes_value(true)
+                .validator(solana_net_utils::is_host)
+                .requires("client_node_id")
+                .help("IP address to use with connection cache"),
+        )
+        .arg(
+            Arg::with_name("client_node_id")
+                .long("client-node-id")
+                .value_name("PATH")
+                .takes_value(true)
+                .requires("json_rpc_url")
+                .validator(is_keypair)
+                .help("File containing the node id (keypair) of a validator with active stake. This allows communicating with network using staked connection"),
```

Comment:
id --> identity. Normally id means identification in computer programs.

Reply:
done